### ISSUE TITLE
RazerS3 binary appears to be broken

### ISSUE THREAD

**Issue by messersc** [bioconda/bioconda-recipes#15384]
```
$ razers3 --help
Illegal instruction
```

See https://github.com/FRED-2/OptiType/issues/90

Downgrading to an older version seems helps. Not sure what the problem is.

**Comment by npavlovikj** [comment]
@messersc , not sure if this helps, but can you please make sure you use the proper order of the conda channels when installing "razers3" (e.g. `-c conda-forge -c bioconda -c defaults` as defined here, https://bioconda.github.io/#set-up-channels)?

I tried installing "razers3" 3.5.3 myself, and everything worked fine:
**(razers3) [fedora@npavlovikj-conda bioconda-recipes]$ razers3 --help
...
VERSION
    Last update:
    razers3 version: 3.5.3 [tarball]
    SeqAn version: 2.1.1**

And here is a list of the dependencies and the respective channels for you to compare with yours:
**(razers3) [fedora@npavlovikj-conda bioconda-recipes]$ conda list
# packages in environment at /anaconda/envs/razers3:
#
# Name                    Version                   Build  Channel
bzip2                     1.0.6                h470a237_2    conda-forge
libgcc-ng                 7.2.0                hdf63c60_3    conda-forge/label/cf201901
libstdcxx-ng              7.2.0                hdf63c60_3    conda-forge/label/cf201901
razers3                   3.5.3                h21aa3a5_2    bioconda
zlib                      1.2.11               h470a237_4    conda-forge**

**Comment by messersc** [comment]
Hi,

that's interesting, although it might not be what causes the crash. I just created a new environment:

```bash
$ conda list
# packages in environment at /fast/users/messersc_c/work/miniconda/envs/razers3:
#
# Name                    Version                   Build  Channel
bzip2                     1.0.6             h14c3975_1002    conda-forge
libgcc-ng                 8.2.0                hdf63c60_1  
libstdcxx-ng              8.2.0                hdf63c60_1  
razers3                   3.5.3                h21aa3a5_2    bioconda
zlib                      1.2.11            h14c3975_1004    conda-forge

$ razers3
Illegal instruction
```

Downgrading the libs to 7.2.0 does not change anything.

```bash
$ conda list
# packages in environment at /fast/users/messersc_c/work/miniconda/envs/razers3:
#
# Name                    Version                   Build  Channel
bzip2                     1.0.6                h470a237_2    conda-forge
libgcc-ng                 7.2.0                hdf63c60_3  
libstdcxx-ng              7.2.0                hdf63c60_3  
razers3                   3.5.3                h21aa3a5_2    bioconda
zlib                      1.2.11               h470a237_4    conda-forge

$ razers3 
Illegal instruction
```

This is on a `Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz`.

**Comment by npavlovikj** [comment]
You are right Clemens, this is specific to your machine and not the channels - I tried using the exact packages from your env and "razers3" worked fine for me...

Looks like there is similar issue reported for bioconda package on Xeon machine, https://github.com/bioconda/bioconda-recipes/issues/14613.
If you have the time, you can try buildling the conda recipe locally with the suggested flags to see if that helps...

**Comment by holtgrewe** [comment]
cf #11790
